src/redux/books/book.js

```diff
@@ -0,0 +1,30 @@
+const bookReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'NEW_BOOK':
+      return [
+        ...state, action.payload,
+      ];
+    case 'DELETE_BOOK':
+      return state.filter((book) => book.id !== action.payload.id);
+    default:
+      return state;
+  }
+};
```

Comment:
- [ ] Good job so far, but you should define `NEW_BOOK` and `DELETE_BOOK` as action types. So it can be something like: `const NEW_BOOK = bookstore/books/NEW_BOOK` and `const DELETE_BOOK = bookstore/books/DELETE_BOOK` 🙃 😉